azurerm/resource_arm_container_group.go

```diff
@@ -166,21 +166,24 @@ func resourceArmContainerGroup() *schema.Resource {
 						"environment_variables": {
 							Type:     schema.TypeMap,
 							Optional: true,
-							ForceNew: true,
+						},
+
+						"sensitive_environment_variables": {
```

Comment:
Can we align with Azure documentation ?
https://docs.microsoft.com/en-us/azure/container-instances/container-instances-environment-variables#secure-values

Reply:
I disagree, as this implies a higher level of security that isn't there since the values are persisted in the HCL file plan test. Since it is not encrypted I would feel better if we would expose it as sensitive and in the code pass it as secureValue as required by the API.

Reply:
I think there was a misunderstanding that this wasn't surfacing something the API here internally. While these variable are not "secure" in Terraform, they are more secure in the container group and that is the name used in the API, so we should probably match it.

Reply:
Agreed, I have reverted back to using the secure name instead of sensitive.